package.json

```diff
@@ -37,6 +37,8 @@
   },
   "license": "Apache 2.0",
   "dependencies": {
+    "element-size": "^1.1.1",
+    "lodash": "^3.9.3",
```

Comment:
I added it in because the build was complaining about it not being there. I don't remember where it was being used though.